includes/class-wc-payments.php

```diff
@@ -649,9 +649,11 @@ public static function init_rest_api() {
 		$tos_controller = new WC_REST_Payments_Tos_Controller( self::$api_client, self::$card_gateway, self::$account );
 		$tos_controller->register_routes();
 
-		include_once WCPAY_ABSPATH . 'includes/admin/class-wc-rest-payments-settings-controller.php';
-		$settings_controller = new WC_REST_Payments_Settings_Controller( self::$api_client, self::$card_gateway, self::$digital_wallets_gateway );
-		$settings_controller->register_routes();
+		if ( WC_Payments_Features::is_grouped_settings_enabled() ) {
```

Comment:
Just ensuring that the controller for REST data is added only when the feature is enabled - otherwise it's not needed